src/annotator/features.js

```diff
@@ -2,15 +2,21 @@ import { TinyEmitter } from 'tiny-emitter';
 
 import { warnOnce } from '../shared/warn-once';
 
+/**
+ * @typedef {import('../types/annotator').FeatureFlags} IFeatureFlags
+ */
+
 /**
  * List of feature flags that annotator code tests for.
  *
  * @type {string[]}
  */
-const annotatorFlags = [];
+const annotatorFlags = ['html_side_by_side'];
 
 /**
  * An observable container of feature flags.
+ *
+ * @implements {IFeatureFlags}
```

Comment:
I don't love the fact that the interface and the main implementation have the same name. The main distinction between the interface and this class is that the interface only provides methods to read flags and subscribe to updates, whereas the class has methods to update them.